playground/cloudflare-env/vite.config.custom-env.ts

```diff
@@ -0,0 +1,10 @@
+import { cloudflare } from '@flarelabs-net/vite-plugin-cloudflare';
+import { defineConfig } from 'vite';
+
+export default defineConfig(() => {
+	process.env = { ...process.env, CLOUDFLARE_ENV: 'custom-env' };
```

Comment:
Does this need to be undone after the call to `cloudflare()` (perhaps in a finally block) to prevent it from leaking into other tests?

Reply:
Hmm. I tried restoring with `finally` but that doesn't work (presumably because it is restored before the variable is read). It's annoying we can't use `vi.stubEnv` (https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/127#discussion_r1914825858) as that would be a lot nicer.

Reply:
If you end up using `package.json` scripts for running the tests then we could just pass the environment variable in there.

Reply:
I think you're right about this potentially leaking. Now that I have the test that reads the `CLOUDFLARE_ENV` variable from a `.env` file I could just remove this variant. What do you think?